litex/build/gowin/apicula.py

```diff
@@ -0,0 +1,46 @@
+from litex.build.generic_platform import *
+from litex.build import tools
+from litex.build.gowin.gowin import _build_cst
+from litex.build.yosys_nextpnr_toolchain import YosysNextPNRToolchain
+
+class GowinApiculaToolchain(YosysNextPNRToolchain):
+    family     = "gowin"
+    synth_fmt  = "json"
+    pnr_fmt    = "report"
+    packer_cmd = "gowin_pack"
+
+    def __init__(self):
+        super().__init__()
+        self.additional_cst_commands = []
+
+    def build_timing_constraints(self, vns):
+        # nextpnr-himbaechel for gowin currently doesnt supports timing constaints
+        pass
```

Comment:
Not required since implemented by `GenericToolchain` module (and must return `("","")` when unused).